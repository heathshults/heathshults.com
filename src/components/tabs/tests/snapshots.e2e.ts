import { snapIt } from '../../../test';

const component = 'tabs';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap(
      'renders correctly with no props',
      `<hs-tabs>
      <hs-tab>This is tab one</hs-tab>
      <hs-tab>This is tab two</hs-tab>
    </hs-tabs>`
    );
    snap(
      'renders correctly with props set',
      `<hs-tabs>
      <hs-tab header="Tab one" open>This is tab one</hs-tab>
      <hs-tab header="Tab two" disabled>This is tab two</hs-tab>
    </hs-tabs>`
    );
  });
});
