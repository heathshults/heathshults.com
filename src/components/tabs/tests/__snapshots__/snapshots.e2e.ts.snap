// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tabs renders renders correctly with no props 1`] = `"<hs-tabs class=\\"hydrated\\"><div class=\\"hs-tabs\\"><div role=\\"tablist\\" class=\\"hs-tabs\\"><div class=\\"hs-tabs__nav\\"><div class=\\"hs-tabs__headings\\"><button role=\\"tab\\" class=\\"hs-tab-heading\\"></button><button role=\\"tab\\" class=\\"hs-tab-heading\\"></button></div></div><hs-tab class=\\"hydrated\\"><div role=\\"tabpanel\\" hidden=\\"\\" class=\\"hs-tabs__tab\\">This is tab one</div></hs-tab><hs-tab class=\\"hydrated\\"><div role=\\"tabpanel\\" hidden=\\"\\" class=\\"hs-tabs__tab\\">This is tab two</div></hs-tab></div></div></hs-tabs>"`;

exports[`tabs renders renders correctly with props set 1`] = `"<hs-tabs class=\\"hydrated\\"><div class=\\"hs-tabs\\"><div role=\\"tablist\\" class=\\"hs-tabs\\"><div class=\\"hs-tabs__nav\\"><div class=\\"hs-tabs__headings\\"><button role=\\"tab\\" class=\\"hs-tab-heading hs-tab-heading--active\\">Tab one</button><button role=\\"tab\\" disabled=\\"\\" class=\\"hs-tab-heading\\">Tab two</button></div></div><hs-tab header=\\"Tab one\\" open=\\"\\" class=\\"hydrated\\"><div role=\\"tabpanel\\" class=\\"hs-tabs__tab\\">This is tab one</div></hs-tab><hs-tab header=\\"Tab two\\" disabled=\\"\\" class=\\"hydrated\\"><div role=\\"tabpanel\\" hidden=\\"\\" class=\\"hs-tabs__tab\\">This is tab two</div></hs-tab></div></div></hs-tabs>"`;
